lib/sql.ml

```diff
@@ -424,7 +424,7 @@ let () =
   ["least";"greatest";"coalesce"] ||> multi_polymorphic;
   "strftime" |> exclude 1; (* requires at least 2 arguments *)
   ["concat";"strftime"] ||> multi ~ret:(Typ Text) (Typ Text);
-  ["date";"time"] ||> monomorphic Text [Datetime];
+  ["date";"time"] ||> monomorphic Datetime [Datetime];
```

Comment:
is it correct for `time`?

Reply:
I'm not sure (didn't see any actual use of TIME). I changed this to leave it as it was.

also
> Extracts the time part of the time or datetime expression expr and returns it as a string.